vsock/src/thread_backend.rs

```diff
@@ -0,0 +1,306 @@
+// SPDX-License-Identifier: Apache-2.0 or BSD-3-Clause
+
+#![deny(missing_docs)]
+
+use super::{
+    rxops::*,
+    vhu_vsock::{
+        ConnMapKey, Error, Result, VSOCK_HOST_CID, VSOCK_OP_REQUEST, VSOCK_OP_RST,
+        VSOCK_TYPE_STREAM,
+    },
+    vhu_vsock_thread::VhostUserVsockThread,
+    vsock_conn::*,
+};
+use log::{info, warn};
+use std::{
+    collections::{HashMap, HashSet, VecDeque},
+    os::unix::{
+        net::UnixStream,
+        prelude::{AsRawFd, FromRawFd, RawFd},
+    },
+};
+use virtio_vsock::packet::VsockPacket;
+use vm_memory::bitmap::BitmapSlice;
+
+pub struct VsockThreadBackend {
```

Comment:
pub(crate) should be enough ? I think that's true for most of the stuff here as this is a binary crate and most of the stuff shouldn't be visible out of this crate. I do see some violation of this in existing crates, will add an issue for that.

Reply:
okay, I'll change